persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConversationDAO.kt

```diff
@@ -12,6 +12,9 @@ data class ConversationEntity(
     val mutedTime: Long = 0,
     val lastNotificationDate: String?,
     val lastModifiedDate: String,
+    // Date that indicates when the user has seen the conversation,
+    // nullable lastReadDate means that the conversation is never seen by the user
+    val lastReadDate: String? = null,
```

Comment:
Regardless of how we store this in SQLite we can convert it to an Instant at this level and above. If this is a big change I'm fine with doing in a separate PR.

Reply:
I think this was discussed in last collective and there is a refactor ticket pending, i'll leave it as it for now but add a todo